### PR TITLE
Publish Microsoft Authored templates to azure samples browser

### PR DESCRIPTION
**Pull request by v-xuto**
Publish `svc-invoke-dapr-csharp` template to azure samples browser.

@rajeshkamal5050 and @hemarina for notification.

### CODE REVIEW

**Review by hemarina — APPROVED** [review]
LGTM. @greenie-msft for notification. Could you review this PR?

**Comment by v-xuto** [comment]
@greenie-msft Could you help review this PR and merge it if it looks good?

**Comment by v-xuto** [comment]
@greenie-msft Could you help review this PR and merge it if it looks good? Thanks in advance.

**Comment by hemarina** [comment]
@tonybaloney Would you mind review this PR?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+page_type: sample
+languages:
+- azdeveloper
+- csharp
+- bicep
+products:
+- azure
+- azure-container-apps
+urlFragment: svc-invoke-dapr-csharp
+name: Microservice communication using service invoke (C#)
```
Comment (tonybaloney) [inline README.md]:
```suggestion
name: Microservice communication using Dapr's Service Invocation API (C#)
```

**Review by tonybaloney — CHANGES_REQUESTED** [review]

**Comment by v-xuto** [comment]
@tonybaloney I have updated by your comments, please re-review.

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**